RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -797,24 +797,37 @@ boolean L12_startWar()
 	// wear the appropriate war outfit based on auto_hippyInstead
 	equipWarOutfit();
 	
+	if (auto_haveCCSC())
```

Comment:
might want to `&& !have_skill($skill[Comprehensive Cartography])`

Reply:
Done.